org.mamedev.MAME.yaml

```diff
@@ -0,0 +1,72 @@
+id: org.mamedev.MAME
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: mame
+separate-locales: false
+finish-args:
+  - --device=all
+  - --persist=.mame
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - name: mame
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+      strip: true
+    build-commands:
+      - make -j ${FLATPAK_BUILDER_N_JOBS}
+      - install -Dm 0755 mame /app/bin/mame
+      - install -Dm 0644 ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 0644 ${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - |
+        for i in 32 48 64 128 256; do
+          install -Dm 0644 mame${i}x${i}.png /app/share/icons/hicolor/${i}x${i}/apps/${FLATPAK_ID}.png
+        done
+    sources:
+    - type: archive
+      url: https://github.com/mamedev/mame/archive/refs/tags/mame0244.tar.gz
+      sha256: 843c917edc46008a27b439a1fc66fc12a27c84e55e08753b963789e5614cebf7
+    - type: file
+      path: mame32x32.png
+    - type: file
+      path: mame48x48.png
+    - type: file
+      path: mame64x64.png
+    - type: file
+      path: mame128x128.png
+    - type: file
+      path: mame256x256.png
+    - type: file
```

Comment:
these icons aren't square. Also only 128x128 is required.

Reply:
The build failed with :

```
error: /srv/buildbot/worker/build-x86_64-2/build/builddir/export/share/icons/hicolor/32x32/apps/org.mamedev.MAME.png is not a valid icon: Expected a square icon but got: 32x11
```

Reply:
This seems like an unusual requirement, but easy one to fix

Reply:
all icons are square. I don't see that being unusual.

Reply:
I've tested only in KDE, but it has no problem centering the rectangular images